### ISSUE TITLE
Theme links in top navigation bars

### ISSUE THREAD

**Issue by MisRob** [learningequality/kolibri#9019]
Due to technical limitations, we couldn't utilize theme colors fully for links in top navigation bars in the past, e.g. Home, Library, and Bookmarks links in Learn:

![links](https://user-images.githubusercontent.com/13509191/150531284-f3ddd52a-16f4-4989-b8e7-6cebaa6afdf5.png)

When a link is active, we use hardcoded `white` colors

https://github.com/learningequality/kolibri/blob/9048d77029e622b57d95009ab8b968f12dd5d99a/kolibri/core/assets/src/views/Navbar/NavbarLink.vue#L111-L112

where we should use [`textInverted` token](https://design-system.learningequality.org/colors/#tokens-textInverted).

It should be now possible to refactor `NavbarLink` to use [router-link scoped slot API ](https://v3.router.vuejs.org/api/#v-slot-api-3-1-0) that's been released meanwhile. That should allow us to use `textInverted` token instead of hardcoded white.

## User-facing consequences

`textInverted` token is used to create sufficient contrast when used on dark backgrounds such as `tokens.primary`. Using hardcoded color instead of the token increases the risk of not having the proper contrast in theme plugins.

**Comment by zhihanC** [comment]
Hi @MisRob ! I'm a new contributor looking to find a good first issue to work on. Is this issue taken?

**Comment by MisRob** [comment]
Hello @zhihanC, thank you, you're welcome to assign yourself. Please note that I've updated the obsolete link to the scoped slot API documentation to https://v3.router.vuejs.org/api/#v-slot-api-3-1-0 in the issue description. I think one way to solve this could be to utilize `isActive` similarly to [this example](https://v3.router.vuejs.org/api/#example-applying-active-class-to-outer-element).

**Comment by zhihanC** [comment]
Thank you @MisRob! The documentation to the API and the example were pretty helpful - I think I have a general idea of how to do it.

By the way, can you add me as a collaborator of this project or point me to someone who can do that? I tried assigning myself to the issue but I don't think I have the permissions to do so!

**Comment by MisRob** [comment]
@zhihanC You're welcome, I hope it goes well. Let us know if you need any support. I've just assigned you to the issue and also assigned myself so that I won't forget to keep an eye on comments. When you open a PR, you're welcome to add yourself [to the list of contributors](https://github.com/learningequality/kolibri/blob/develop/AUTHORS.md).

**Comment by amitpanwar789** [comment]
@MisRob @bjester    can i work on this issue?

**Comment by marcellamaki** [comment]
Hi @amitpanwar789! Thank you for your interest in contributing! I've done a little bit of refactoring work on the App bar in recent months, so I'm going to review and this issue as necessary to make sure it's fully up to date and ready to work on. It should be ready by next week. After that, we would welcome your contributions! I'll assign us both to the issue so we can keep track of it, and then I'll follow up with you early next week to confirm it should be ready to work on. 😊

**Comment by marcellamaki** [comment]
Hi @amitpanwar789 - confirming that I've reviewed the current version of `NavbarLink.vue` and the scope of this issue is up to date and it should be ready to work on! 

If you have questions as you are working, please add a comment here and tag me in it, and I will be working reduced hours over the upcoming holidays and New Year, but I will get back to you as soon as possible. Thanks again for offering to contribute to Kolibri :)

**Comment by amitpanwar789** [comment]
should I start working from develop or release-v0.15.x branch ?@marcellamaki

**Comment by marcellamaki** [comment]
Hi @amitpanwar789 - you can work from `develop`. Please note that if this is your first time running Kolibri, you will have to do your setup in the UI on the `release-v0.15.x` branch, as this feature in _develop_ is currently unstable. You can see more in the [Getting Started Documentation](https://kolibri-dev.readthedocs.io/en/develop/getting_started.html). After you have set up a facility in the UI, you can stop the kolibri server, switch branches to `develop`, re-install your dependencies, and work from there, as the facility data will persist.

The [Kolibri Design System documentation about theme colors](https://design-system.learningequality.org/colors/#tokens-textInverted) that is linked in the original issue description will also be helpful, so be sure to look that over before you begin.
